<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>25</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="PyDMPushButton" name="reverse_button">
     <property name="toolTip">
      <string/>
     </property>
     <property name="PyDMToolTip" stdset="0">
      <string>Tweak ${SIG} in the negative direction.</string>
     </property>
     <property name="channel" stdset="0">
      <string>haven://${SIG}</string>
     </property>
     <property name="PyDMIcon" stdset="0">
      <string>minus</string>
     </property>
     <property name="pressValue" stdset="0">
      <string>0</string>
     </property>
     <property name="relativeChange" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="value_spin_box">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="toolTip">
      <string>How much to tweak the signal with each tweak.</string>
     </property>
     <property name="maximum">
      <double>100000000000000000620008645040778319495168.000000000000000</double>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PyDMPushButton" name="forward_button">
     <property name="toolTip">
      <string/>
     </property>
     <property name="PyDMToolTip" stdset="0">
      <string>Tweak ${SIG} in the positive direction.</string>
     </property>
     <property name="channel" stdset="0">
      <string>haven://${SIG}</string>
     </property>
     <property name="PyDMIcon" stdset="0">
      <string>plus</string>
     </property>
     <property name="pressValue" stdset="0">
      <string>0</string>
     </property>
     <property name="relativeChange" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyDMPushButton</class>
   <extends>QPushButton</extends>
   <header>pydm.widgets.pushbutton</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
